govcd/vm.go

```diff
@@ -482,10 +486,12 @@ func (vm *VM) attachOrDetachDisk(diskParams *types.DiskAttachOrDetachParams, rel
 	}
 
 	// Send request
-	req := vm.client.NewRequest(nil, http.MethodPost, *reqUrl, bytes.NewBufferString(xml.Header+string(xmlPayload)))
-
-	req.Header.Add("Content-Type", execLink.Type)
-
+	reqPayload := bytes.NewBufferString(xml.Header + string(xmlPayload))
+	req := vm.client.NewRequest(nil, http.MethodPost, *reqUrl, reqPayload)
+	req.Header.Add("Content-Type", attachOrDetachDiskLink.Type)
+	util.Logger.Printf("[DEBUG] POSTING TO URL: %s", reqUrl)
+	util.Logger.Printf("[DEBUG] XML TO SEND:\n%s", reqPayload)
```

Comment:
All information regarding request and response is logged automatically so no need explicitly log that. You can see implementation in logging.go

Reply:
This log messages are removed.